java-checks/src/test/files/checks/StaticMethodCheck.java

```diff
@@ -0,0 +1,58 @@
+package foo;
+
+class Utilities {
```

Comment:
could you add examples with usages of `super` and `this` to also show that these cases  are covered?